prog/mutation.go

```diff
@@ -92,6 +92,8 @@ func (p *Prog) Mutate(rs rand.Source, ncalls int, ct *ChoiceTable, corpus []*Pro
 							a.Val ^= 1 << uint64(r.Intn(64))
 						}
 					}
+				case *LenType:
+					r.mutateSize(arg.(*ConstArg), *parent)
```

Comment:
There is a `p.Target.assignSizesCall(c)` call below, which will undo this mutation, as far as I understand.

Reply:
Done